README.md

```diff
@@ -2,25 +2,36 @@
 
 Drought-related food insecurity in Chad, Somalia, Ethiopia and Malawi. 
 
+## Getting started
```

Comment:
Note: we haven't reached consensus as a team whether to use venvs and whether conda is the right tool. but leave it in